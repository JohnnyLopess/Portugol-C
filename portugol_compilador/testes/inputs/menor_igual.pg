programa  
{  
    inicio
        inteiro c = 1  
        se( 10 <= 5){
            c++
        }  
}  