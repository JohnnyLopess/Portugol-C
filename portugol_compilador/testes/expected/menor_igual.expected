#include <stdio.h>

int main() {
    int c = 1;
    if (10 <= 5) {
        c++;
    }
    return 0;
}